components/StatusMarker/index.js

```diff
@@ -4,21 +4,33 @@ export default function StatusMarker({ part }) {
   return (
     <>
       {part.inAssembler === true ? (
-        <StatMarker style={{ backgroundColor: "var(--color-inAssembler)" }}>
+        <StatMarker
+          key={"inAssembler"}
+          style={{ backgroundColor: "var(--color-inAssembler)" }}
```

Comment:
Don't know what `StatMarker` is. Is is a styled-components? Then you could move the style in the styled-component.

Reply:
StatMarker is a reusable component, but the backgroundColor is set depending on if the status like `inAssembler` is set true